tests/test_.py

```
@@ -0,0 +1,232 @@
+    """
+
+    @pytest.fixture(autouse=True)
+    def _ensure_views(self) -> None:
+        if not views_mod:
+            pytest.skip("Views module not importable.")
+
+    @pytest.mark.parametrize(("path", "expected_status"), [
+        ("/health/", 200),
+        ("/api/v1/resource/", 200),
+    ])
+    def test_get_endpoints_happy_paths(self, api_client: Any, path: str, expected_status: int):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_get_endpoints_happy_paths`

Add return type annotation: `None`

(ANN201)